apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/ir/Fragment.kt

```diff
@@ -3,14 +3,16 @@ package com.apollographql.android.compiler.ir
 import com.apollographql.android.compiler.Annotations
 import com.apollographql.android.compiler.ClassNames
 import com.apollographql.android.compiler.SchemaTypeSpecBuilder
+import com.squareup.javapoet.CodeBlock
 import com.squareup.javapoet.FieldSpec
 import com.squareup.javapoet.TypeSpec
+import java.util.*
 import javax.lang.model.element.Modifier
 
 data class Fragment(
     val fragmentName: String,
     val source: String,
-    val typeCondition: String,
+    val possibleTypes: List<String>,
```

Comment:
that is the idea, @martijnwalraven mentioned that on iOS they don't use `typeCondition ` at all, so we can say `possibleTypes` is mandatory